wsit/bundles/webservices-osgi-aix/pom.xml

```diff
@@ -290,19 +291,28 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <additionalOptions>
-                        --add-exports java.security.jgss/sun.security.krb5=ALL-UNNAMED
-                        --add-exports java.xml/com.sun.org.apache.xerces.internal.impl.dv.util=ALL-UNNAMED
-                        --add-exports java.xml/com.sun.org.apache.xerces.internal.dom=ALL-UNNAMED
-                        --add-exports java.xml/com.sun.org.apache.xerces.internal.parsers=ALL-UNNAMED
-                    </additionalOptions>
                     <excludePackageNames>*.msv.*:xsdlib.*:*com.sun.xml.ws*:com.sun.tools.*:com.sun.enterprise.*:java.*:isorelax.*:src.*:*javanet*:*test*:*ctc*:*glassfish*:*com.sun.istack.tools*:javax.xml.crypto.*:org.apache.xml.security.*:org.apache.jcp.xml.dsig.internal.*:org.apache.commons.*:com.sun.grizzly.*</excludePackageNames>
                     <additionalDependencies>
                         <dependency>
                             <groupId>org.glassfish.hk2</groupId>
                             <artifactId>hk2-api</artifactId>
                             <version>2.5.0</version>
                         </dependency>
+                        <dependency>
+                            <groupId>org.glassfish.grizzly</groupId>
+                            <artifactId>grizzly-framework</artifactId>
+                            <version>2.4.4</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>org.glassfish.grizzly</groupId>
+                            <artifactId>grizzly-portunif</artifactId>
+                            <version>2.4.4</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>org.apache.tomcat</groupId>
+                            <artifactId>tomcat-coyote</artifactId>
+                            <version>9.0.8</version>
+                        </dependency>
```

Comment:
Why not to place these dependencies to some common parent pom and reference them just from here? I saw that you have added the same dependencies in webservices-osgi module also.

Reply:
that's going to happen in the next set of changes